src/EditorFeatures/Next/IntelliSense/Completion/Presentation/VisualStudio15CompletionSetFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Presentation
+{
+    [ExportVersionSpecific(typeof(ICompletionSetFactory), VisualStudioVersion.VS15)]
+    internal sealed class VisualStudio15CompletionSetFactory : ICompletionSetFactory
+    {
+        public ICompletionSet CreateCompletionSet(
+            CompletionPresenterSession completionPresenterSession,
+            ITextView textView,
+            ITextBuffer subjectBuffer)
+        {
+            return new FilteredRoslynCompletionSet(
```

Comment:
I think i'd call these more Dev15CompletionSet and Dev14CompletionSet.  Make those types explicit that they differ to provide these different levels of support.  it also means a clean naming system moving forward.